src/server/w32_svr_periodic_hook.c

```diff
@@ -402,7 +439,7 @@ main(int argc, char *argv[])
 		fprintf(stderr, "Out of memory\n");
 		return 1;
 	}
-
+	CLEAR_HEAD(svr_allresvs);
```

Comment:
svr_allresvs is accessed inside get_resv_list () and if we do not initialize here, it will get a crash while executing (resc_resv *)GET_NEXT(svr_allresvs) from get_resv_list().